### DISCUSSION TITLE
Why not have star meaning required?

### DISCUSSION THREAD

**Discussion by losthank** [ant-design/ant-design#28967] [Q&A]
![image](https://user-images.githubusercontent.com/29891556/105326739-0992aa80-5c09-11eb-93a8-676a339de5f0.png)
We can see there is no Star that means Required from the example in Antd4.
The detail scene is that when we use Form.Item, we write a <span> or any other html tag after <Input> or other html tag wrapped by <Form.Item>, like thefirst picture and corresponding code in antd example, after adding "Need Help", the star vanished.
![image](https://user-images.githubusercontent.com/29891556/105327238-9dfd0d00-5c09-11eb-9dcb-589ce4360871.png)

**Comment by xrkffgg** [comment]
https://ant.design/components/form-cn/#components-form-demo-required-mark
